server/tserver/src/main/java/org/apache/accumulo/tserver/compactions/CompactionManager.java

```diff
@@ -91,144 +89,10 @@ public ExtCompInfo(KeyExtent extent, CompactionExecutorId executor) {
     }
   }
 
-  private class Config {
```

Comment:
This is a nice change, breaking out this class and giving it a more proper class name.